supplementary_style_guide/style_guidelines/links.adoc

```diff
@@ -2,8 +2,95 @@
+
+=== Technical best practices
+
+* Include links only when necessary.
+* Use hyperlinks and use them consistently.
+* Default to followable, crawlable links. Do not use “nofollow” links unless absolutely necessary.
+* Do not use JavaScript links.
++
+Example JavaScript:
+----
+<script type="text/javascript" src="/path/to/script.js"></script>
+----
+* Do not use Flash links.
++
+Example Flash:
++
+----
+on(release)
+{
+getURL("http://www.example.com", "_blank");
+}
+----
+* Do not use links in Frames or iframes.
++
+Example iFrame:
++
+----
+ <iframe src="url" title="description"></iframe> 
+----
+
+=== Link text best practices
+
+* Contextually describe what the user will find at the target location so that they can decide if they want to leave their current location.
+* Use a concise sentence or sentence fragment as the link text.
+* Do not use bare URLs. Bare URLs are unhelpful because they do not provide adequate context about the link target.
+* Avoid irrelevant link text.
+
+
+
+
 [[internal-links]]
 == Internal links
 
+Internal links are links within \www.redhat.com.
+
+.Example AsciiDoc: Internal links
+----
+For more information about <topic>, see xref:<link>[<link_text>].
+----
+
+To link to another guide or to a user story within the same guide:
+----
+For more information about <topic>, see link:<link>[<link_text>] in _<title_name>_.
+----
+
+[[rh-kb-links]]
+=== Linking to Red Hat Knowledgebase articles
+
+* Ensure that your product team supports linking to Knowledgebase articles.
+* Use the title of the Knowledgebase article for the link text, or use descriptive running text.
+* When not using running text, indicate that the link target is a  Knowledgebase article.
+* When the link appears in *Additional resources*, put the article title first, followed by `(Red Hat Knowledgebase)`.
+
+
+.Example: Article title
+
+For a non-cloud environment, you can resize the disk and file system. For more information, see the Red Hat Knowledgebase solution link:https://access.redhat.com/solutions/199573[Does RHEL 7 support online resize of disk partitions?].
+
+.Example: Running text
+
+If your Apache web server configuration enables SSL security, enable only the TLSv1 protocol and disable SSLv2 and SSLv3. This configuration is required because of the link:https://access.redhat.com/solutions/1232413[POODLE SSL vulnerability (CVE-2014-3566)].
+
+.Example: Additional resources
+
+* link:https://access.redhat.com/solutions/199573[Does RHEL 7 support online resize of disk partitions?] (Red Hat Knowledgebase)
+
+
+
 [[external-links]]
 == External links
+
+External links are links to a domain other than \www.redhat.com.
+
+* Avoid links to external sites if possible. External links can change or be unreliable. 
+* Avoid deep links in particular (to a specific page or image from another company) to prevent an ongoing maintenance responsibility and to prevent bypassing a website's legal information.
```

Comment:
Can "in particular" be removed? I don't see how it adds value, unless I'm misreading the sentence